### PR TITLE
Experiments tracking

### PR DESCRIPTION
**Pull request by daytime-em**
This should be a Draft until:

* MuxCore is released with the Session Data changes
* The code has been reviewed and I've added support for Session Data to earlier versions of this SDK

### CODE REVIEW

File: MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java [hunk 1]
```diff
@@ -337,6 +347,52 @@ public void onAdsManagerLoaded(AdsManagerLoadedEvent adsManagerLoadedEvent) {
...
+  protected List<SessionTag> parseHlsSessionData(List<String> hlsTags) {
+    List<SessionTag> data = new ArrayList<>();
+    for(String tag : filterHlsSessionTags(hlsTags)) {
+      data.add(parseHlsSessionTag(tag));
+    }
+    return data;
+  }
+
+  protected SessionTag parseHlsSessionTag(String line) {
+    Matcher dataId = RX_SESSION_TAG_DATA_ID.matcher(line);
```
Comment (nbirkenshaw-mux) [inline MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java:378]: Is there a situation where the data ID is valid and the value is not? (And vice versa?) And if they're both in the same line wouldn't it be easier to extract both values using a single regex match?

**Review by nbirkenshaw-mux — APPROVED** [review]

File: MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java (hunk 1 above)
Comment (daytime-em) [reply MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java:378]: The RFC says that `DATA-ID` is required but `VALUE` is not. A `DATA-ID` can also be associated with a `URI`, which this parser doesn't support.

ref: https://datatracker.ietf.org/doc/html/rfc8216#section-4.3.4.4

Comment (nbirkenshaw-mux) [reply MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java:378]: Thanks! Is that just because URIs are not of any interest?

Comment (daytime-em) [reply MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java:378]: Yes. For the experiments feature, we're not adding any tags with `URI`s, just strings containing analytics dimensions.

As much as I would love to make a general-purpose `m3u8` parser, it feels hard to justify the business cost to me.

File: build.gradle [hunk 2]
```diff
@@ -66,7 +66,7 @@ allprojects {
         releaseVersion = 'r2.1.1'
         releaseWebsite = 'https://github.com/google/ExoPlayer'
 
-        muxCoreVersion = "7.0.10"
+        muxCoreVersion = "dev-session-data-event-e5d0aff"
```
Comment (skidder) [inline build.gradle]: Do you plan to change this as part of a release? (not sure if this was accidentally committed)

Comment (daytime-em) [reply build.gradle]: I was going to update it before moving this PR out of Draft, and after releasing a MuxCore with the required features.

**Pull request merged**